File: ModuleDaily/src/main/java/module/DailyApp.java
package module;

import com.qihuan.commonmodule.base.BaseApp;
import com.qihuan.commonmodule.net.Api;

import me.jessyan.retrofiturlmanager.RetrofitUrlManager;

/**
 * DailyApp
 *
 * @author Qi
 */
public class DailyApp extends BaseApp {

    @Override
    public void onCreate() {
        super.onCreate();
        RetrofitUrlManager.getInstance().putDomain(Api.ZHIHU_DOMAIN_VALUE, Api.ZHIHU_BASE_URL);
    }
}

File: module_daily/src/main/java/module/DailyApp.kt
package module

import com.qihuan.commonmodule.base.BaseApp
import com.qihuan.commonmodule.net.Api

import me.jessyan.retrofiturlmanager.RetrofitUrlManager

/**
 * DailyApp
 *
 * @author Qi
 */
class DailyApp : BaseApp() {

    override fun onCreate() {
        super.onCreate()
        RetrofitUrlManager.getInstance().putDomain(Api.ZHIHU_DOMAIN_VALUE, Api.ZHIHU_BASE_URL)
    }
}
